ui/raidboss/data/05-shb/trial/titania-ex.js

```diff
@@ -232,36 +175,13 @@
     },
     {
       id: 'TitaniaEx Fae Light',
-      regex: Regexes.startsUsing({ id: '3D2C', source: 'Titania', capture: false }),
-      regexDe: Regexes.startsUsing({ id: '3D2C', source: 'Titania', capture: false }),
-      regexFr: Regexes.startsUsing({ id: '3D2C', source: 'Titania', capture: false }),
-      regexJa: Regexes.startsUsing({ id: '3D2C', source: 'ティターニア', capture: false }),
-      regexCn: Regexes.startsUsing({ id: '3D2C', source: '缇坦妮雅', capture: false }),
-      regexKo: Regexes.startsUsing({ id: '3D2C', source: '티타니아', capture: false }),
-      alertText: function(data) {
-        if (data.role == 'tank' || data.role == 'healer') {
-          return {
-            en: 'Tank Buster',
-            de: 'Tankbuster',
-            fr: 'Tankbuster',
-            ja: 'タンクバスター',
-            cn: '死刑',
-            ko: '탱버',
-          };
-        }
-      },
-      infoText: function(data) {
-        if (data.role != 'tank' && data.role != 'healer') {
-          return {
-            en: 'Tank Cleave',
-            de: 'Tank Cleave',
-            ja: 'タンクバスター',
-            fr: 'Tank cleave',
-            cn: '坦克顺劈',
-            ko: '탱버',
-          };
-        }
-      },
+      regex: Regexes.startsUsing({ id: '3D2C', source: 'Titania' }),
+      regexDe: Regexes.startsUsing({ id: '3D2C', source: 'Titania' }),
+      regexFr: Regexes.startsUsing({ id: '3D2C', source: 'Titania' }),
+      regexJa: Regexes.startsUsing({ id: '3D2C', source: 'ティターニア' }),
+      regexCn: Regexes.startsUsing({ id: '3D2C', source: '缇坦妮雅' }),
+      regexKo: Regexes.startsUsing({ id: '3D2C', source: '티타니아' }),
+      response: Responses.tankBuster(),
```

Comment:
This loses the tank cleave portion here.  Can you make this tankBuster for tanks/healers and say "cleave" for non-tanks/non-healers so that they don't stand in front?